sdk/react/src/markers/MarkersProgressBarOverlayContainer/MarkersProgressBarOverlayContainer.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
+
+import MarkerProgressBarOverlay from '../MarkerProgressBarOverlay';
+import styles from './MarkersProgressBarOverlayContainer.styles';
+import type { Marker as MarkerType } from '../../types/Marker';
+
+type Props = {
+  accentColor?: ?string,
+  duration: number,
+  markers: Array<MarkerType>,
+  style?: ViewStyleProp,
+};
+
+const MarkersProgressBarOverlayContainer = ({
+  accentColor, duration, markers, style,
+}: Props) => (
+  markers.length > 0
+    ? (
+      <View style={[styles.root, style]}>
```

Comment:
What do you think about extract this large block in method?

Reply:
This is a functional component which actually contains only the ternary operator returning this large block, so there is no point in extracting it into a separate component.